src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -558,6 +558,27 @@ def test_ppmu_source(multi_instrument_session):
     multi_instrument_session.pins['site0/LO0', 'site1/HI0'].ppmu_source()
 
 
+def test_clock_generator_abort(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_abort()
+
+
+def test_clock_generator_generate_clock(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_generate_clock(
+        1e6,
```

Comment:
Use a float here

Reply:
1e6 is a float.
Run `type(1e6)` and you'll see.